GitExtensions.ruleset

```diff
@@ -48,9 +48,11 @@
     <Rule Id="SA1403" Action="None" />
     <Rule Id="SA1404" Action="None" />
     <Rule Id="SA1413" Action="None" />
+    <Rule Id="SA1503" Action="None" />
     <Rule Id="SA1504" Action="None" />
     <Rule Id="SA1512" Action="None" />
     <Rule Id="SA1516" Action="None" />
+    <Rule Id="SA1519" Action="None" />
```

Comment:
Why are disabling these rules?

Reply:
Seems like not needed.
Removing

Reply:
Fixed by adding braces as expected by the rule: https://github.com/gitextensions/gitextensions/pull/8156/files#diff-0c448da7cf150a550546e895757d16c2R182